doc/ChangeLog.md

```diff
@@ -1594,7 +1594,7 @@ Supported YANG models in addition to those used by sysrepo and netopeer:
  - N/A
 
 [buildroot]:  https://buildroot.org/
-[UNRELEASED]: https://github.com/kernelkit/infix/compare/v25.05.1...HEAD
```

Comment:
I usually keep (and update) the UNRELEASED link, preparing for the next. It doesn't mess up any formatting in the generated output, so it's safe to keep.